VeganVirus/VeganVirus/FruitThrow.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+#include "Draw.h"
+#include "MouseManager.h"
+#include <vector>
+
+#define FRUIT_GRAVITY 500
+// pixels per second^2
+
+#define CURSOR_MASS 1
+#define FRUIT_MASS 10
+
+#define BOUNCE_FACTOR 2.5
+
+#define RADIUS_RANGE 40
+#define BOUNCE_MARGIN_X 4
+#define BOUNCE_MARGIN_Y -4
+
+typedef struct
+{
+	double x;
+	double y;
+} DPoint_t;
+
+class FruitThrow
+{
+public:
+	// FruitThrow Constructor
+	// input: pointer to draw object, mouse manager reference
+	FruitThrow(Draw* draw, MouseManager& mouseManager);
+
+	// FruitThrow Destructor
+	~FruitThrow();
+
+	// method updates the fruit throw object
+	// input: time since last update
+	// return: none
+	void update(double dt);
+
```

Comment:
Comments to your functions in this file are not consistent in spaces with other files where you put a lot of spaces after what the function returns.

Reply:
Doesn't really matter. The function documentation doesn't have to be exactly the same for everyone who works on the projects, small deviations are fine.